### ISSUE TITLE
[Documentation] "Edit this page on GitHub!" isn't so useful (anymore?)

### ISSUE THREAD

**Issue by escapedcat** [vuetifyjs/vuetify#10058]
### Environment

**Browsers:** Chrome 79.0.3945.88
**OS:** Mac OS 10.14.6

### Steps to reproduce
For example
1. https://vuetifyjs.com/en/getting-started/quick-start
2. Click "Edit this page on GitHub!"  
     (https://github.com/vuetifyjs/vuetify/tree/master/packages/docs/src/data/pages/getting-started/QuickStart.json)
3. Be confused as a new user because instead of looking at an expected markdown file the user finds a "weird" json file

### Expected Behavior
Find a markdown-file. If this sin't wanted or possible maybe try to provide some help how certain elements on the page can be edited via Github.

### Actual Behavior
Not the expected content but a "weird" json file.

### Reproduction Link
<a href="https://vuetifyjs.com/en/getting-started/quick-start" target="_blank">https://vuetifyjs.com/en/getting-started/quick-start</a>

### Other comments
I had to search the string I wanted to update in the whole repo to see where it was. Not a big deal but maybe it could help other users to add some info into the json where the files with content are actually located.  

In other projects this "Edit at Github"-links lead to a simple markdown file which can also be updated by less experienced people. 

It's ok to close this issue. Just wanted to give a heads up how this might feel for other users.

**Comment by MajesticPotatoe** [comment]
JSON file is in fact the page structure. I'm open to suggestions on verbiage that will clarify without being as wordy. 
May just be better to remove the portion all together, as in fact can potentially be multiple files that need to be edited to modify any given page (structure, lang, example, snippet).

**Comment by escapedcat** [comment]
Hm, I kinda like still like the link to some sort of more direct file even if it's a json. Then I don't need to search which repo it is etc.  

On the other other hand users might expect something more simple.  
Comments to the file-path to each entry would be helpful. But json doesn't allow comments. 

It's hard to understand that `bash_vue_cli_create_app` means `packages/docs/src/snippets/bash/vue_cli_create_app.txt`.  

Not sure what is possible in this setup you use. Maybe fake json comment-entries? Not a very good solution I guess.

**Comment by MajesticPotatoe** [comment]
Might just be better to update the contribution guide with those nuances and update that snippet at the bottom to navigate to that. Would be a better platform for clarification anyways. Comments are just gonna muddy the heck out of things, and much more overhead and information duplication. Contribution guide is in need of a cleanup/update anyways.

**Comment by escapedcat** [comment]
You're right. I could add something to this part for exmaple: https://vuetifyjs.com/en/getting-started/contributing#pull-requests-for-docs

**Comment by MajesticPotatoe** [comment]
There's actually a lot of info that could be added to it that's not there in terms of structure/what things go where / how it all ties together which would be beneficial to newcomers, which is why I was gonna do it, was waiting for it to settle down as we've changed structure of the docs and how some of it works a couple times in the past year. 

I think we've settled pretty nicely on the structure now, and with our push toward v3 would probably be beneficial. I will see if i can focus some of my time to it this weekend.
